### DISCUSSION TITLE
Sampling strategies are not applied to the image 'jaeger:all-in-one'

### DISCUSSION THREAD

**Discussion by nfsp3k** [jaegertracing/jaeger#2696] [Q&A] [answered]
**Describe the bug**
Sampling strategies are not applied to jaeger all-in-one.
Even though I set the default sampling rate to 0.05, the Jaeger samples every trace.

**To Reproduce**
1. Create a Kubernetes cluster and run an application in the cluster
2. Create jaeger:all-in-one pod with the sampling strategies argument in the cluster
3. Send traffic to the cluster and see if the sampling strategies are applied.

**Expected behavior**
The sampling rate I mentioned in the sampling strategies json file is applied to the jaeger in my cluster.

**Screenshots**
My jaeger .yaml file is as follows.
![image](https://user-images.githubusercontent.com/4262433/102595990-db2e3500-415b-11eb-910f-e03fa15ad658.png)

My JSON file is as follows.
![image](https://user-images.githubusercontent.com/4262433/102596033-ec774180-415b-11eb-9866-b2df4167ca94.png)

The partial output of the command 'kubectl describe pod jaeger-xxx' is as follows:
![image](https://user-images.githubusercontent.com/4262433/102596191-2b0cfc00-415c-11eb-8ea1-3986e2c6eff0.png)

The Jaeger pod works correctly as follows.
![image](https://user-images.githubusercontent.com/4262433/102596266-44ae4380-415c-11eb-9edf-40a1179f2e2b.png)

**Version (please complete the following information):**
 - OS: Ubuntu 16.04
 - Jaeger version: I don't know... How can I get that? I'm using the image whose URL is jaegertracing/all-in-one:latest.
 - Deployment: Kubernetes

**What troubleshooting steps did you try?**
I even don't know what I can do about that.

**Comment by jpkrohling** [comment]
Sorry for the noise, I thought this was about the operator.

**Reply by nfsp3k** [reply]
Thanks for the comment, @jpkrohling.
I'm going to see the operator repository.

**Comment by yurishkuro** [comment, marked as answer]
Did you configure all-in-one to use remote sampler? By default it uses always-on sampler.

https://github.com/jaegertracing/jaeger/blob/6d315d791709cba7d814aca842ad57ba42969303/cmd/all-in-one/main.go#L252-L261

You need to set `JAEGER_SAMPLER_TYPE=remote` and `JAEGER_SAMPLER_PARAM={default probability}` env vars.
